plugin/adsk/scripts/mayaUsd_preferenceTab.mel

```diff
@@ -392,10 +408,22 @@ global proc mayaUsd_PrefHoldState(string $mode)
 	};
 	prefsHoldOptionVars($prefOptionVars, $mode); 
 
-	if ($mode == "remove") {
-		// Finalize preference changes into MayaUsd.
+    if ($mode == "save") {
+        // Preferences dialog was opened, make sure we don't have this transient optionVar.
+        optionVar -remove mayaUsd_PrefUntexturedModeChanged;
+    }
+
+	if ($mode == "restore") {
+        if (`optionVar -q mayaUsd_PrefUntexturedModeChanged`) {
+            // Need to refresh the viewport for display mayaUsd_ShowDisplayColorTextureOff
+            ogs -reset;
```

Comment:
Should we remove the `mayaUsd_PrefUntexturedModeChanged` optionVar once the viewport is reset?